### DISCUSSION TITLE
What is the correct way to use google fonts?

### DISCUSSION THREAD

**Discussion by FDiskas** [resend/react-email#1674] [Q&A]
Looking for a good example of using custom fonts with latin extended version

https://github.com/resend/react-email/blob/6b12e0a4a32dd6807d4138681fccafd35069a985/apps/web/components/_components/layout.tsx#L21-L30

As I can see in this example layout - **the usage is wrong** 
because the font URL points to CSS rather than to the font it self.

**Comment by gabrielmfern** [comment]
You're absolutely right this layout has that wrong, thanks for pointing that out. One good example is the exact one from [the docs](https://react.email/docs/components/font#getting-started)
